locksmith/src/routes/v2/metadata.ts

```diff
@@ -50,4 +51,10 @@ router.put(
   updateUserMetadata
 )
 
+router.get(
+  '/locks/:lockAddress/users/:userAddress',
```

Comment:
so...  I realize I gave you a wrong indication. We should include the network here because it is the lock's network, not the user's network.

`/v2/api/metadata/${network}/locks/${lockAddress}/users/${address}`